rollrus.go

```diff
@@ -75,11 +97,10 @@ func (r *Hook) Fire(entry *log.Entry) (err error) {
 
 // Levels returns the logrus log levels that this hook handles
 func (r *Hook) Levels() []log.Level {
-	return []log.Level{
-		log.ErrorLevel,
-		log.FatalLevel,
-		log.PanicLevel,
+	if r.firedLevels == nil {
```

Comment:
Is this actually ever true? Maybe the Hook itself should be un-exported, making the only way to set this up is via the SetupLogging* functions above and this can then be removed?